android/src/main/java/cloud/pace/plugins/cloudsdk/CloudSDK.kt

```diff
@@ -52,11 +49,8 @@ class CloudSDK : Plugin() {
         val callAuthenticationMode = call.getString(AUTHENTICATION_MODE) ?: WEB
         val callEnvironment = call.getString(ENVIRONMENT) ?: PRODUCTION
 
-        val authenticationMode = try {
-            AuthenticationMode.valueOf(callAuthenticationMode.toUpperCase(Locale.getDefault()))
-        } catch (e: Exception) {
-            AuthenticationMode.WEB
-        }
+        val authenticationMode = if (callAuthenticationMode.equals("native", true))
+            AuthenticationMode.NATIVE else AuthenticationMode.WEB
```

Comment:
Please put the `else` on the same line as the `if`, otherwise this syntax will be haunting me at night...